src/Rule/Callback.php

```diff
@@ -18,6 +19,10 @@
 use Yiisoft\Validator\WhenInterface;
 
 /**
+ * Defines validation options to validating the value using a callback.
```

Comment:
Maybe add a little bit more explanation? For example that it can be used as an alternative for creating custom rules with handlers - simpler, but not reusable, etc.

Reply:
Also the separate usage / usage for a class method can be mentioned too.

Reply:
I think that belongs to readme.